ui/imports/shared/popups/UserStatusContextMenu.qml

```diff
@@ -70,7 +70,7 @@ PopupMenu {
             hoverEnabled: true
             cursorShape: Qt.PointingHandCursor
             onClicked: {
-                openProfilePopup(profileModel.profile.username, profileModel.profile.pubKey, profileModel.profile.thumbnailImage || "");
+                openProfilePopup(profileModel.profile.username, profileModel.profile.pubKey, profileModule.thumbnailImage || "");
```

Comment:
Some of these have already been addressed in: https://github.com/status-im/status-desktop/pull/3961

@iurimatias fee free to close mine in favour of this

Reply:
ups... I tested that one first and already merged, seeing this one now

Reply:
@PascalPrecht @iurimatias are all issues addressed then? shall this one be closed?